test/test-remote/looker/src/args.ts

```diff
@@ -215,8 +233,9 @@ export function parseCmdlineArgs(): void {
   parser.add_argument("--change-streams-every-n-cycles", {
     help:
       "Use the same log/metric stream for N cycles, then create a new set of " +
-      "streams (unique label sets). Default: new streams are created with every " +
-      "write/read cycle. For log streams, when a new stream is initialized it " +
+      "streams (unique label sets), or <=0 to reuse streams for the process lifetime. " +
+      "Default: new streams are created with every write/read cycle (1). " +
+      "For log streams, when a new stream is initialized it " +
```

Comment:
Thanks!

Reply:
future: we should not allow negative values and document what `0` means : )

Reply:
switched to require >= 0